src/testdrive/src/main.rs

```diff
@@ -50,7 +50,7 @@ struct Args {
     /// Named AWS region to target for AWS API requests.
     #[structopt(long, default_value = "localstack")]
     aws_region: String,
-    /// Custom AWS endpoint.
+    /// Custom AWS endpoint
```

Comment:
This seems errant! I was careful to consistently use periods in all the other docstrings.

Reply:
Yup, I added a description of the default, and then when I deleted the default description I accidentally deleted the dot.